docs/reference/module-api/module-overview.md

```diff
@@ -257,14 +257,231 @@ modules.export = {
 };
 ```
 
-## Customization functions
+## Initialization function
 
 ### `async init(self)`
-### `beforeSuperClass(self)`
+
+This function runs once when the Apostrophe app first start up. It takes the module, as `self`, as an argument.
```

Comment:
first starts up.

Reply:
Based on years and years of developer emails and questions, I think it's worth adding "it does NOT run on every request. For that, see event handlers, specifically `@apostrophecms/page:beforeSend`."

Reply:
Many new Apostrophe developers don't really get that node servers are long-lasting processes.

Reply:
Good point. This isn't specifically about pages, so I don't think we should point to that specifically, but we can link over to an event handlers guide.

Reply:
Adding "To run code on every request, or in other situations when the app is running, see the event handlers documentation." Let me know if there's other language for that you think would be more appropriate.